build.gradle.kts

```diff
@@ -431,9 +431,9 @@ subprojects {
     reports.html.outputLocation.set(file("${rootProject.projectDir}/build/reports/"))
     val skipTests = project.hasProperty("skipTests")
     if (!skipTests) {
-      jvmArgs = listOf("-Xmx4G")
+      val extraArgs = project.property("extraJvmArgs") as List<String>
+      jvmArgs = listOf("-Xmx4G") + extraArgs
```

Comment:
Can you provide relevant screenshots to prove that this parameter actually works?

Reply:
![Uploading image.png…]()

Reply:
![WeChat627be692d957c301004fa7a165a7ec33](https://github.com/user-attachments/assets/e84c2a82-3da8-4f1c-8bc7-52463f1e47cb)

Reply:
Should this be backported to branch-0.6?

Reply:
Need backport